### PR TITLE
Add CC-BY-SA-3.0-IGO

### LINKED ISSUE #1895: New license request: CC-BY-SA-3.0-IGO
### Introduction

Variant of the "CC-BY-3.0-IGO" license that was [somewhat recently included](https://github.com/spdx/license-list-XML/pull/1547). Like with most Creative Commons' licenses, there exist a "Share Alike" version that requires derivative work to be released under the same license. 

### License Name

Creative Commons Attribution-ShareAlike 3.0 IGO

### Suggested short identifier

CC-BY-SA-3.0-IGO

### URL to license text

https://creativecommons.org/licenses/by-sa/3.0/igo/legalcode

### OSI Status

I don't know

### License author or steward

Creative Commons

### URL to project(s) that use license

-

### PR DESCRIPTION
**Pull request by Doctor-love**
Fixes #1895

Signed-off-by: Joel Rangsmo <joel@rangsmo.se>

### CODE REVIEW

**Comment by Doctor-love** [comment]
Seems like you need to approve the workflow to execute, @swinslow

**Comment by jlovejoy** [comment]
@Doctor-love - this need the text .txt file as well, see step 2 here https://github.com/spdx/license-list-XML/blob/main/DOCS/request-new-license.md#accepted-license-process
:)

I will wait to run tests, as they will fail if I run them now

**Comment by Doctor-love** [comment]
You're absolutely right, @jlovejoy - added the missing file! :-)

File: src/CC-BY-SA-3.0-IGO.xml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
...plied; (iii) to the extent reasonably practicable, the URI, if any, that the Licensor specifies to be associated with the Work, unless such URI does not refer to the copyright notice or licensing information for the Work; and, (iv) consistent with Section 3(b), in the case of an Adaptation, a credit identifying the use of the Work in the Adaptation. The credit required by this Section 4(c) may be implemented in any reasonable manner; provided, however, that in the case of an Adaptation or Collection, at a minimum such credit will appear, if a credit for all contributors to the Adaptation or Collection appears, then as part of these credits and in a manner at least as prominent as the credits for the other contributors. For the avoidance of doubt, You may only use the credit required by this Section for the purpose of attribution in the manner set out above and, by exercising Your rights under this License, You may not implicitly or explicitly assert or imply any connection with, sponsorship or endorsement by the Licensor or others designated for attribution, of You or Your use of the Work, without the separate, express prior written permission of the Licensor or such others.</item>
```
Comment (goneall) [inline src/CC-BY-SA-3.0-IGO.xml:65]: @jlovejoy You have 2 copies of `c.` which is causing it to fail

**Review by goneall — CHANGES_REQUESTED** [review]

File: src/CC-BY-SA-3.0-IGO.xml (hunk 1 above)
Comment (Doctor-love) [reply src/CC-BY-SA-3.0-IGO.xml]: It's like the tests exist for a reason - thanks for the catch!

**Pull request merged**